model/model.go

```diff
@@ -677,4 +698,63 @@ type (
 		Message *string     `json:"message"`
 		Error   *ErrorData  `json:"error"`
 	}
+
+	// CancelPayoutRequest request schema for cancel payout
+	CancelPayoutRequest struct {
+		BulkPayoutID string `json:"payout_id"`
+		Reason       string `json:"reason"`
+	}
+
+	// BulkPayoutConfig schema for payout config
+	BulkPayoutConfig struct {
+		Provider                 string  `json:"provider"`
```

Comment:
create a `payout.go` file in the model package to house all payout related models.

Reply:
Alright sir